rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -152,6 +155,8 @@ function(rapids_cpm_nvcomp)
   # provide consistent targets between a found nvcomp and one building from source
   if(NOT TARGET nvcomp::nvcomp AND TARGET nvcomp)
     add_library(nvcomp::nvcomp ALIAS nvcomp)
+    add_library(nvcomp::nvcomp_gdeflate ALIAS nvcomp_gdeflate)
```

Comment:
We should have these in separate logic blocks to guard against them going away in the future

Reply:
Done.